### ISSUE TITLE
Converting a webhook start event into another event type does not unlink the element template

### ISSUE THREAD

**Issue by chillleader** [camunda/camunda-modeler#3537]
### Describe the bug

It is not possible to remove the webhook connector using the wrench ("Change element" menu) once applied.

_(originally reported by @christian-konrad in https://github.com/camunda/team-connectors/issues/377)_

### Steps to reproduce

1. Start modeling a process
2. Create a webhook connector start event
3. Change element type of the start event, e.g. to message start event
4. Verify that the element template is still linked

### Expected behavior

Similarly to other connectors, template should be unlinked, but the extension properties should be retained.
In other words, this should be equivalent to unlinking the element template.

## Camunda Modeler system information
 * Version: 5.9.0
 * Operating System: Mac OS 10.15.7 
 * Plugins: <no plugins>
 * Execution Platform: BPMN - Camunda Platform 8

### Additional context

https://user-images.githubusercontent.com/56470362/229083882-231412ea-b029-44d5-a144-d6d31c64829d.mov

Related to https://github.com/bpmn-io/bpmn-js-properties-panel/issues/642

**Comment by barmac** [comment]
Thanks for reporting. I can reproduce it.

**Comment by barmac** [comment]
I am looking into this now as it's getting even more annoying with the message event connectors.
